packer/conf/buildkite-agent/systemd/buildkite-agent@.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Buildkite Agent (%i)
+Documentation=https://buildkite.com/agent
+After=syslog.target
+After=network.target
+After=docker.service
+
+[Service]
+Type=simple
+User=buildkite-agent
+Environment="HOME=/var/lib/buildkite-agent"
```

Comment:
You might like to add `USER=buildkite-agent`, too. Maybe there are others?

Reply:
Done!